tests/vectors/nem/models/blocks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
not sure if it makes sense to have file with empty array vs no output file if empty?